.github/workflows/test_pull_requests.yml

```diff
@@ -93,10 +93,23 @@ jobs:
           - python: 3.9
             platform: ubuntu-latest
             backend: pyside6
+          # pyside 6
           - python: '3.10'
             platform: ubuntu-latest
             backend: pyside6
-
+          - python: '3.11'
```

Comment:
Might be a bit clearer to use [`exclude`](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#excluding-matrix-configurations) here to exclude the Pyside2 3.11 build?

Reply:
Yes, that's a good idea. Once the tests are passing I can try to do that.